src/alembic/versions/0decc0e420e7_add_model_file_name_to_robot_type.py

```diff
@@ -0,0 +1,36 @@
+"""add model file name to robot type
+
+Revision ID: 0decc0e420e7
+Revises: b5f91e296db4
+Create Date: 2022-07-22 15:45:42.585508
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0decc0e420e7'
+down_revision = 'b5f91e296db4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('generationtemplate', 'version',
+               existing_type=sa.REAL(),
+               type_=sa.FLOAT(precision=2),
+               existing_nullable=False)
```

Comment:
What is happening here? Why does the data type change?

Reply:
No idea tbh. The same thing seemed to happen with all migration scripts once the version was added.

Reply:
Weird. Well, tests are passing, and it does not see too harmful. Not enough time left to debug the root of the issue for now :(